.yamato/config/universal.metafile

```diff
@@ -89,6 +89,28 @@ test_platforms:
         flags:
           - [all]: --timeout=3000
           - <reuse_blocks.cache_server>
+
+  - type: Standalone
+    name: Standalone_XR_cache
+    commands:
+      - [Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.connect_bokken>
+      - [Win_DX11, Win_DX12, Win_Vulkan, Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.turn_off_firewall>
+      - [Win_DX11, Win_DX12, Win_Vulkan]: cd Tools && powershell -command ". .\Unity.ps1; Set-ScreenResolution -width 1920 -Height 1080"
+      - [all]: <shared.platform.commands.setup_utr>
+      - [OSX_Metal]: <shared.platform.commands.unity_downloader_cli>
+      - [all]: <shared.platform.commands.call_utr>
+        flags:
+          - [all]: --extra-editor-arg="-xr-reuse-tests"
+          - <reuse_blocks.cache_server>
```

Comment:
This line can be safely removed, as done in this PR https://github.com/Unity-Technologies/Graphics/pull/3541/files#diff-74d0ff2e643e59221524764c520ab17911cff3edaabe1635f6eb39602003988aL36.
We only need to pass the server cache arguments at the build step, no need at the run step.

Reply:
@theo-pnv Thanks!  May I remove the extraneous line in my next PR?  If it is not hurting anything, then I'd rather get this PR merged without doing another test round, since Yamato has been having troubles lately.  Also, are you able to merge this now that all reviewers have approved, or is it someone else... or is further action needed from me?

Reply:
Ah -- I saw your message in Slack just now.  I'll remove the extraneous line and re-launch the test pipelines this afternoon (California time).  Hopefully enough of them will finish and pass during your work day (Europe time) for you to merge.  I assume that if the tests reveal problems after I remove the one line, then you could just merge commit 475e026.

Reply:
@theo-pnv 
In Standalone_XR_cache I removed “- <reuse_blocks.cache_server>” from commands flags (but left it intact for commands_build flags) in both universal.metafile and universal_foundation.metafile.  Then I generated YML files and launched the Yamato pipelines below.  These results may be compared against the master pipeline links above.  

If commit  aea8197 has questionable results, or if they have not finished during your work day, then I’d be happy if you merged earlier commit 475e026.  Thanks in any case! 


* Nightly Universal - trunk (tests/xr/urp_standalone_dennisd_3 aea8197)
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/tests%252Fxr%252Furp_standalone_dennisd_3/.yamato%252Fall-universal.yml%2523Nightly_Universal_trunk/5416048/job

* Universal PR Job - trunk  (tests/xr/urp_standalone_dennisd_3 aea8197)
https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/tests%252Fxr%252Furp_standalone_dennisd_3/.yamato%252Fall-universal.yml%2523PR_Universal_trunk/5416020/job